apps/docs/content/components/Text/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Text
+description: Use the text component to render body content
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image displaying each text from size 100 to 700.](https://github.com/primer/brand/assets/912236/67d89861-7a1e-4df9-a801-01d11f4dfb17)
```

Comment:
```suggestion
![The text component has size ranges from 100 to 700 with 100 being the smallest, and 700 being the largest.](https://github.com/primer/brand/assets/912236/67d89861-7a1e-4df9-a801-01d11f4dfb17)
```

I think it might make more sense to just describe what the image is showing, or not have alt text at all